third_party/ABSEIL_MANUAL.md

```diff
@@ -0,0 +1,30 @@
+# Abseil in gRPC
+
+This document explains how to use Abseil throughout gRPC. Note that this isn't
+supposed to explain general usage of Abseil.
+
+## The version of Abseil
+
+gRPC is inteded to use the LTS versions of Abseil only because it simplfies
```

Comment:
gRPC is inteded -> gRPC intends

simplfies -> simplifies

Reply:
Done :)